stable/telegraf/values.yaml

```diff
@@ -11,6 +11,8 @@ image:
 
 podAnnotations: {}
 
+serviceAnnotations: {}
```

Comment:
nit: Can you put this under `service.annotations` ?

Reply:
Sure can. I was debating doing that myself.